target/linux/x86/base-files/etc/board.d/01_leds

```diff
@@ -16,6 +16,13 @@ pc-engines-apu1|pc-engines-apu2|pc-engines-apu3|pc-engines-apu4|pc-engines-apu5|
 	ucidef_set_led_netdev "lan" "LAN" "apu:green:2" "br-lan"
 	ucidef_set_led_default "diag" "DIAG" "apu:green:1" "1"
 	;;
+sophos-sg-105wr1|sophos-sg-125wr1|sophos-sg-135wr1|sophos-xg-105wr1|sophos-xg-125wr1|sophos-xg-135wr1)
+	ucidef_set_led_netdev "wlan" "WiFi" "ath9k-phy0" "phy0tpt"
+	;;
+sophos-sg-105wr2|sophos-sg-125wr2|sophos-sg-135wr2|sophos-xg-105wr2|sophos-xg-125wr2|sophos-xg-135wr2|
```

Comment:
Missing `\`?

Reply:
Thanks, I've tried to fix it, but not sure why bcm47xx files were added to this commit.